### PR TITLE
nctl: rework ejection health check

### PR DESCRIPTION
**Pull request by TomVasile**
Changes:
- switched out the way we detect ejected nodes from the logs
    - the log message no longer contains `.fields.finalized_block` so it was switched to derive it from `.fields.participation`

### CODE REVIEW

**Review by rafal-ch — APPROVED** [review]
LGTM :+1: 

Should work as expected, even though `log_participation()` is based on a timer (`log_participation_interval = '1min'`), whereas `finalized_block` was logged immediately.

Please double-check with @fizyk20 maybe?

**Comment by TomVasile** [comment]
@fizyk20 Do you know of a better way I can check for ejected validators based on the logs? I'm open to reworking this if needed.

Maybe we log this out specifically to make it easier to check?

**Comment by TomVasile** [comment]
labeling as do not merge, in standup we decided to log out the ejections when the switch block hits, will use that once merged

**Pull request closed without merging**
